### PR TITLE
Increase time out before rejecting with timeout

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: projects/engine/src/plugin/abstract.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class Plugin<T extends Api = any, App extends ApiMap = any> implements Pl
           if (this.requestQueue.length !== 0) this.requestQueue[0]()
         }
 
-        const ref = setTimeout(() => { timedout = true, letcontinue() }, 10000)
+        const ref = setTimeout(() => { timedout = true, letcontinue() }, 60000)
```
Comment (GrandSchtroumpf) [inline projects/engine/src/plugin/abstract.ts:86]: 1min looks super long. What's your usecase ?

Comment (yann300) [reply projects/engine/src/plugin/abstract.ts:86]: running the unit test

Comment (GrandSchtroumpf) [reply projects/engine/src/plugin/abstract.ts:86]: Maybe adding a "enableTimeout = false" property in the engine would be a better fix then I think

Comment (yann300) [reply projects/engine/src/plugin/abstract.ts:86]: a bit more precision: this is for calling the solidity unittest plugin. it takes often more than 10s for having a result.
> Maybe adding a "enableTimeout = false" property

ok right, but we need timeout anyway , what about adding the opt `timeOut` (in ms) to the engine?

Comment (GrandSchtroumpf) [reply projects/engine/src/plugin/abstract.ts:86]: Sounds like a good idea

Comment (GrandSchtroumpf) [reply projects/engine/src/plugin/abstract.ts:86]: Sounds like a good idea

**Comment by GrandSchtroumpf** [comment]
@yann300 Can we close it ? It should be solved by #203

**Pull request closed without merging**